eng/pipelines/common/evaluate-changed-paths.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
```

Comment:
Can we add a comment here indicating the overall purpose of this script?

Reply:
Sure.

Reply:
Fixed.